sdk/keyvault/Azure.Security.KeyVault.Secrets/samples/Sample1_HelloWorld.md

```diff
@@ -0,0 +1,96 @@
+# Creating, getting, updating, and deleting secrets
```

Comment:
I think it would be better to have an md file for both the synchronous and asynchronous samples and have them cross-reference each other. If people agree we should add an issue to track adding these.

Reply:
That's a lot to maintain. What if, instead, we reference both sync and async sample files by relative path?

Reply:
I updated azure/azure-sdk#769 to discuss.